v3/integrations/logcontext-v2/nrwriter/example/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"log"
+	"os"
+	"time"
+
+	"github.com/newrelic/go-agent/v3/integrations/logcontext-v2/nrwriter"
+	"github.com/newrelic/go-agent/v3/newrelic"
+)
+
+const (
+	loggerFlags = log.Ldate | log.Ltime
+)
+
+func main() {
+	app, err := newrelic.NewApplication(
+		newrelic.ConfigAppName("nrwriter log writer example"),
+		newrelic.ConfigFromEnvironment(),
+		newrelic.ConfigInfoLogger(os.Stdout),
+		newrelic.ConfigAppLogDecoratingEnabled(true),
+	)
+	if err != nil {
+		panic(err)
+	}
+
+	app.WaitForConnection(5 * time.Second)
+
+	nrwriter := nrwriter.New(os.Stdout, app)
```

Comment:
I can see it might be a little confusing to make a local variable with the same name as the `nrwriter` package.